INSTALL.md

```diff
@@ -0,0 +1,85 @@
+# BUILDING FROM SOURCE
+
+By contributing to this project you agree to the [Code of Conduct](https://coralproject.net/code-of-conduct.html).
+
+## Requirements
+
+* [Node 7.2.1](https://nodejs.org/es/download/package-manager)
+* Mongo 3.4.0
+* Redis 3.2.6
+
+### Operating System
+
+- We recommend hosting [Talk](https://github.com/coralproject/talk) with any flavor of Linux
+- 1GB memory (minimum)
+- 5GB storage (minimum)
+
+## Initial Setup
+
+### Get the code
+
+Clone the repository:
+
+`
+git clone https://github.com/coralproject/talk
+`
+
+### Installation
+
+* cd talk
+* npm install
+* npm run build
```

Comment:
The build step is already added to the `postinstall` hook, shouldn't re-run it. Also, enclose in a triple backtick instead of a list.